lgc/state/PalMetadata.cpp

```diff
@@ -969,3 +920,179 @@ void PalMetadata::eraseFragmentInputInfo() {
   if (array2It != m_pipelineNode.end())
     m_pipelineNode.erase(array2It);
 }
+
+// =====================================================================================================================
+// Returns the calling convention of the first shader stage that will be executed in the pipeline.
+unsigned PalMetadata::getCallingConvenctionForFirstHardwareShaderStage() {
+  const unsigned numberOfShaderStages = 5;
```

Comment:
nit: prefer `constexpr` for values that have to be known at compile time

Reply:
done